packages/terra-form-select/src/terra-dev-site/test/form-select/MaxHeight.test.jsx

```diff
@@ -2,11 +2,92 @@ import React from 'react';
 import classNames from 'classnames/bind';
 import Select from '../../../Select';
 import styles from './common/Select.test.module.scss';
+import SelectField from '../../../SelectField';
+import ComboboxField from '../../../ComboboxField';
+import SearchSelectField from '../../../SearchSelectField';
+import MultiSelectField from '../../../MultiSelectField';
+import TagSelectField from '../../../TagSelectField';
 
 const cx = classNames.bind(styles);
 
 const MaxHeight = () => (
   <div className={cx('content-wrapper')}>
+    <MultiSelectField label="Multi Select Field" placeholder="Select a number" selectId="Multi-Select" defaultValue="1" maxHeight={300}>
```

Comment:
None of these added variants are being tested in the WDIO test. These can be removed.

Reply:
Are you suggesting to move each component into separate tests? I am confused on your comment as I am only trying to test Height variant.

Reply:
These variants were added to an existing test, but the test run wasn't updated. So it is still only testing the previous height. There was a change to the screenshot, but no new functionality is being tested.

![image](https://user-images.githubusercontent.com/6720991/79126466-97319780-7d65-11ea-852c-6efd1eb427c3.png)


![image](https://user-images.githubusercontent.com/6720991/79126503-ab759480-7d65-11ea-9d41-798b40d36b68.png)

The max height on the added variants isn't being tested in this test

Reply:
Got it thank you you will add more tests